### DISCUSSION TITLE
Github Copilot X

### DISCUSSION THREAD

**Discussion by Yuvarajkumar** [community/community#54368] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Dear Team,

I hope this message finds you in good health and high spirits. I am writing to you today to request access to Github Copilot X. I cannot express enough how much I am in need of this innovative tool, and how much it would mean to me to have access to it.

As someone who has been eagerly waiting on the waiting list for this powerful tool, I am excited to explore its features and the possibilities it brings. I strongly believe that this tool will significantly enhance my workflow and productivity, ultimately leading to better output.

I would be eternally grateful if you could grant me access at this time. With your help, I would be able to provide valuable feedback to help improve the development of this tool, and I am more than willing to offer my support in any way possible.

Thank you very much for your kind consideration, and I sincerely hope that you can help me with this request. I look forward to hearing from you soon and am excited about the possibility of being granted access.

Best regards,
Yuvaraj Kumar

**Comment by Akash1134** [comment, marked as answer]
Hi @Yuvarajkumar ,

Thank you for showing interest in Copilot X. While we are unable to guarantee access to feature(s) in the previews, you can [submit your interest by joining the waitlist](https://github.com/features/preview/copilot-x) to preview the features that you are interested in. Joining one waitlist does not confirm your interest in more than one feature, so you must join each separately. Please note we currently don’t have guaranteed timelines for access. 

Once you get access to preview a feature, please keep in mind that what you are getting access to is considered beta or a technical preview, meaning the feature may still have some kinks to work out.

**Reply by JensPanis** [reply]
@Akash1134 Is it possible to gain access to Copilot X when you are Microsoft Gold Partner?

**Comment by PUCHAJDA** [comment]
Hi Team,

Is there any timeline for Copilot X production release. ? Could you provide any time range - whether it's a matter of 2 months or a year ? Depending on this we will consider staying with Copilot licenses or migrate to Tabnine - since there's no sense of using copilot with codex to then optimize the code in gpt4 commercial license. There's a lack of functionality of copilot on codex comparing to copilot x or gpt4. Just let me know that's the plan. I'm also on Copilot X wait list from the beginning - I've also asked our sales contact for the pre license for X without luck.

Kind Regards,
Przemek Puchajda
